### ISSUE TITLE
Training advise on parameter tuning

### ISSUE THREAD

**Issue by MaWentzien** [LTH14/fractalgen#16]
Thanks for sharing your ideas and code.

I am training to train your model from scratch on images from a different domain, mainly microscopy images. Do you have any advise on which params to tune in which order and magnitude? 

For starters I already found it helpful to increase the base learning rate by one or two magnitudes to decrease the loss from ~10 to ~6 while the IS and FID metrics stay ~300. As one training run takes around two days with my dataset, it will take some time to experiment with all the parameters and ranges.

Additionally,
(i) how do the parameter values transfer between models, like fractalmar_large_in256  and fractalmar_base_in256?
(ii) if my dataset does not require classes, how would setting class_num = 1 influence the metrics calculations, and what else should I consider in this case?

Greatly appreciate ideas for a training recipe.

**Comment by LTH14** [comment]
Thanks for your interest.
1. For the IS and FID metrics, the current codebase uses pre-computed ImageNet statistics. You might need to change that to your own dataset's statistics.
2. We didn't spend a lot of time tuning the parameters. Most parameters just follow the previous MAR paper. The large/base just specifies the model size. The training parameters remain the same.
3. Be careful about classes -- during inference, we assume that you are having class labels and therefore will randomly sample class label for you. If you never use it during training, you should set it to the same behavior of your training.
